src/desdemona/query.clj

```diff
@@ -35,16 +58,28 @@
        (finally
          (in-ns (ns-name old-ns)))))))
 
+(def dsl-literal?
+  "Is this a literal in the DSL?"
+  string?)
+
 (defn ^:private dsl->logic
   "Given a DSL query, compile it to the underlying logic (miniKanren)
   expressions."
   [dsl-query]
   (m/match [dsl-query]
-    [((= ((attr lvar) :seq) value) :seq)]
-    `(l/featurec ~lvar {~attr ~value})
-
-    [((= value ((attr lvar) :seq)) :seq)]
-    `(l/featurec ~lvar {~attr ~value})
+    [(('= & terms) :seq)]
```

Comment:
I should consider testing the negative case where some of the terms are bogus, seeing what the error message is, and seeing if I can improve it by introducing a predicate on the terms.